docs/detections/shared-exception-lists.asciidoc

```diff
@@ -0,0 +1,131 @@
+
+* (Optional) *Close all alerts that match this exception and were generated by this rule*:
+Closes all alerts that match the exception's conditions and were generated only by the current rule.
++
+. Click *Add rule exception*.
+
+[float]
+[[link-shared-exception-lists]]
+=== Associate shared exception lists with rules
+
+Apply shared exception lists to rules:
+
+. Go to *Manage* -> *Rule Exceptions*.
+. Open the shared exception list details page by clicking on a shared exception list name. 
+. Click *Manage rules*. 
+. Choose rules to add the exception to. 
++
+TIP: If you know a rule's name, you can enter it into the search bar.
+. Click *Save*. 
+. (Optional) To verify that shared exception list was added to the rules you selected:
+
+.. Open one of the rule’s details page (*Manage → Rules → <Rule name>*).
+.. Scroll down the page, and then select the *Rule exceptions* tab. 
+.. Navigate to the exception items that are included in the shared exception list. The *Affects shared list* link displays in the exception details. Click the link to view the associated shared exception lists.
++
+[role="screenshot"]
+image::images/associated-shared-exception-list.png[Associated shared exceptions]
+
+[float]
+[[view-shared-exception-lists]]
+=== View and filter exception lists 
+
+The Rule Exceptions page displays each shared exception list on an individual row, with the most recently created list at the top. Each row contains details about the shared exception list, such as the:
+
+* Shared exception list name
+* Date the list was created
+* Username of the user who created the list
+* Number of exception items in the shared exception list
+* Number of rules the shared exception list affects
+
+To view the details of an exception item within a shared exception list, expand a row.
+
+[role="screenshot"]
+image::images/view-filter-shared-exception.png[Associated shared exceptions]
+
+To filter exception lists by a specific value, enter a query in the https://elastic.github.io/eui/#/forms/search-bar[Search bar]. You can search the following attributes:
+
+* `name`
+* `list_id`
+* `created_by`
+
+If no attribute is selected, the app searches the list name by default.
+
+[float]
+[[manage-exception-lists]]
+=== Manage shared exception lists
+
+You can edit, export, import and delete shared exception lists from the Rule Exceptions page.  
+
+NOTE: Exception lists that were created in 8.5 and earlier are imported as shared exception lists. 
+
+To export or delete an exception list, select the required action button on the appropriate list. Note that:
+
+* Exception lists are exported to `.ndjson` files.
+* Exception lists are also exported as part of any exported detection rules configured with exceptions. Refer to <<import-export-rules-ui>>.
+* If an exception list is linked to any rules, you'll get a warning asking you to confirm the deletion. If no rules are linked, the list is deleted without confirmation.
```

Comment:
```suggestion
* If an exception list is linked to any rules, you'll get a warning asking you to confirm the deletion. 
```

Reply:
Agree that we can remove this - it's pretty obvious 😅